mkdocs/docs/api.md

```diff
@@ -1129,6 +1129,28 @@ with table.manage_snapshots() as ms:
     ms.create_branch(snapshot_id1, "Branch_A").create_tag(snapshot_id2, "tag789")
 ```
 
+## Table Statistics Management
+
+Manage table statistics with operations through the `Table` API:
+
+```python
+# To run a specific operation
+table.update_statistics().set_statistics(snapshot_id, statistics_file).commit()
+# To run multiple operations
+table.update_statistics()
+  .set_statistics(snapshot_id1, statistics_file1)
+  .remove_statistics(snapshot_id2)
+# Operations are applied on commit.
```

Comment:
nit: add `.commit()` instead of the comment

Reply:
or use `snapshot_id=1`

Reply:
I added the commit() and kept the comment consistent with the other examples.